toketmaster/src/client.py

```diff
@@ -0,0 +1,19 @@
+from algosdk.v2client import algod, indexer
+
+
+_algod_token = "LGlsGlaSiY37Y3xKQly071AsDwS6PeP73mAaFJLF"
+_headers = {
+        "X-API-Key": _algod_token,
+    }
+
+
+algod_client = algod.AlgodClient(
```

Comment:
This is is already built into https://github.com/algocertified/sandbox/blob/master/toketmaster/fixtures.py#L25.

Reply:
I am using the client in the code, as how it is set up right now it is a fixture that I have to request in a test case, which is not entirely what I want.